.travis.yml

```diff
@@ -9,7 +9,7 @@ matrix:
   -  os: osx
 
 go:
-  - 1.13.x
+  - 1.15.x
```

Comment:
(Note that this AFAICS only affects the macOS builds; Linux builds are driven by `Dockerfile` == what `golang-bin` package is installed by `fedora:latest`. But that does end up being 1.15 currently.)